src/CSSList/CSSList.php

```diff
@@ -41,7 +41,7 @@ abstract class CSSList implements Renderable, Commentable
     protected $comments = [];
 
     /**
-     * @var array<int, RuleSet|CSSList|Import|Charset>
+     * @var array<int<0, max>, RuleSet|CSSList|Import|Charset>
```

Comment:
Could this be a `list`?

Reply:
Unfortunately not: [In `remove()`, we `unset()` elements in the array](https://github.com/MyIntervals/PHP-CSS-Parser/blob/main/src/CSSList/CSSList.php#L324), which creates gaps in the keys.